test/controllers/trajectories_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class TrajectoriesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```

Comment:
I think it's time you start writing some tests, what do you think @mtopete1996 ?

Reply:
as you se for, me no problem

Reply:
After the MVP